Source/WebSocket/Websocketpp/x509_cert_utilities.hpp

```diff
@@ -49,6 +49,10 @@
 #include <wincrypt.h>
 #endif
 
+#if HC_PLATFORM == HC_PLATFORM_LINUX
```

Comment:
We should avoid platform specific code. Does including this break other platforms?

Reply:
I can only test it against windows at the moment, and that works.